synapse/lib/stormtypes.py

```diff
@@ -6496,6 +6528,15 @@ async def _methCronMod(self, prefix, query):
         await self.dyncall('cortex', todo, gatekeys=gatekeys)
         return iden
 
+    async def _methCronMove(self, prefix, view):
+        cron = await self._matchIdens(prefix, ('cron', 'set'))
+        iden = cron['iden']
+
+        todo = s_common.todo('moveCronJob', self.runt.user.iden, iden, view)
```

Comment:
we've been moving away from this dyncall stuff ( which only existed for the now-removed spawn option ) so you can just directly call methods off the cortex now with `self.runt.snap.core` ( here and elsewhere )

Reply:
Ah. I didn't know that. Cool :+1: